unlock_indicator.c

```diff
@@ -261,7 +301,7 @@ xcb_pixmap_t draw_image(uint32_t *resolution) {
     timeinfo = localtime(&rawtime);
 
     if (unlock_indicator &&
-        (unlock_state >= STATE_KEY_PRESSED || auth_state > STATE_AUTH_IDLE || show_clock)) {
```

Comment:
I've got a concern with this part - I know that at one point in a merge commit, I forgot to include this and it screwed up a few use cases. I won't be able to test these changes today (everything else looks fine), but I'll make sure that existing usecases should be fine / won't be impacted.